test/client/pages/account/components/settings/PasswordForm.js

```diff
@@ -62,15 +62,15 @@ lab.experiment('Account Settings Password Form', function () {
         });
         var form = TestUtils.renderIntoDocument(FormEl);
 
-        form.setProps({
-            data: mockStoreState
-        });
+        FormEl = React.createElement(Form, Object.assign({}, form.props, { data: mockStoreState }));
```

Comment:
Because setProps is deprecated and is throwing warnings! The last [commit](https://github.com/jedireza/aqua/pull/89/commits/e5d85bf8f752cf866fd4ba5982abd7f0c37b7983) in `test/client/pages/admin/components/admins/CreateNewForm.js` suited best to test the setProps changes instead of above referenced diff.